src/Tribe/Editor.php

```diff
@@ -377,6 +377,7 @@ public function register_blocks() {
 	 * @return bool
 	 */
 	public function is_events_post_type() {
+		return true; // @todo remove
```

Comment:
Temporarily bypassing checks. This is so the existing blocks assets load, and our new block is compiled in the same `app/main.js` file.

I assume we will want to keep that logic here and just adjust these checks to include template pages?